src/PartielsPy/export_configs/_export_config.py

```diff
@@ -0,0 +1,34 @@
+"""An abstract class for export configuration"""
+
+
+class ExportConfig:
+    """Export configuration class
+
+    This class is used to configure the export of the document.
+    It is used to set the adapt option.
+    This class is inherited by the other ExportConfig classes:
+    - ExportConfigImage
+    """
+
+    def __init__(self):
+        self.__adapt = False
```

Comment:
I think a more explicit term should be better, at least `adaptive`? The context is different from the CLI, so you can use longer terms or even several words.